odsclient/core.py

```diff
@@ -1,13 +1,16 @@
+import json
```

Comment:
Please do not re-import `json` here but instead add `dumps` to lines 9 and 12

Reply:
Thanks, I've missed this one. Updated.